### DISCUSSION TITLE
Dificuldades para iniciar na área do desenvolvimento web

### DISCUSSION THREAD

**Discussion by RicardoBaltazar** [frontendbr/forum#1977] [Discussão]
Olá pessoal, estudo desenvolvimento web a quase dois anos mas estou com dificuldades de entrar no mercado. Vocês chegaram a passar por alguma dificuldade no início da carreira? chegaram a ficar perdidos nos estudos? Meu foco é frontend e já estou aprendendo react, e no backend eu sei fazer alguns CRUD's em php mas não é meu foco. Aqui em Porto Alegre tem muitas vagas de php e tbm de frontend. Vocês tem alguma dica, algo que vocês fizeram lá no inicio e que ajudou na carreira? :)

**Comment by AmandaFonseca** [comment]
Olá @RicardoBaltazar  eu  comecei trabalhando numa empresa pequena como estagiária no call center, e o pessoal do meu trabalho nunca gostou de mim acho pq eu não aceitei ser chamada de estagiotária igual os outros e pq lá era uma panelinha familiar, tipo nepotismo tia que empregou sobrinho, sobrinho que empregou o cunhado, esse cunhado empregou os amigos e por ai vai. Enfim no final quem deixou o sistema rodar usando asp clássico em pleno 2019 e deixou o sistema quebrar não foi eu kkk Ai mostrei o meu trabalho de front para o dono e ele me deu uma promoção o que me ajudou foi:

1- Focar numa área
2- Pesquisar as ferramentas e as linguagens que mais aparecem nas vagas de emprego dentro da área que eu escolhi
3- Fiz alguns trabalhos freelances, ganhei mais experiência, fiz amizades e tirei a má impressão que tinha sobre a profissão
4- Cursos da Udemy, Alura e as video aulas dos nossos amigos Indianos kkk

 E agora to fazendo uns freelas o que tbm ajuda a sair da mesmice e te obriga a sempre aprender algo novo.

**Reply by RicardoBaltazar** [reply]
nossa que massa!!! no fim deu tudo certo :) .  obrigado pela dica, realmente tenho receio de focar só em frontend e acabar não conseguindo nenhum emprego por faltar conehcimentos de backend, mas sempre vejo o pessoal falando pra focar em uma coisa que uma hora a gente consegue.

**Comment by ghost** [comment]
Amanda,

Como você começou a conseguir trabalhos de freelancer? Se outros colegas
quiserem dar dicas também de como começar eu agradeço.

Em qui, 14 de jan de 2021 21:33, AmandaFonseca <notifications@github.com>
escreveu:

**Comment by ninetails** [comment]
Tanto teu GitHub e LinkedIn estão bem massa.  Talvez valeria ver outros perfis bem famosos pra pegar umas idéias pra ter o que melhorar em relação a _pitch_ pessoal.

Eu diria pra vc separar teus estudos e empregabilidade. O que é mais prioritário para você: aprender algo novo e consolidar aprendizado ou encontrar uma vaga? Pois se for o segundo, você pode focar teus estudos de acordo com o que é pedido pelas vagas nos lugares que você tenha maior desejo de entrar.

Também diria ao mesmo tempo você sempre ampliar teu networking. Procurar grupos no Discord e Meetups, participe deles, troque contatos... Além de aprender algo novo, você se aproxima mais de outros devs e com isso também de recrutadores pelo LinkedIn.

**Comment by DyogoViana** [comment]
Podem tentar deixar mais visual tbm o portifa.

saca o meu depois, eu tô tentando apresentar de forma visual pq nem sempre é uma pessoa que saca apenas os códigos no recrutamento.

**Comment by sarahdialmeida** [comment]
Dyogo,
Concordo que é uma boa estratégia apresentar também visualmente. Vi que
você usou o Behance pro portfolio e achei bem legal! Tem alí uma interação
de comunidade?

Em sex., 15 de jan. de 2021 às 19:52, Dyogo Viana <notifications@github.com>
escreveu:
